packages/lib/schedules/client/transformers.ts

```diff
@@ -37,8 +37,11 @@ export function transformAvailabilityForClient(schedule: ScheduleWithAvailabilit
 
 export function transformDateOverridesForClient(schedule: ScheduleWithAvailabilities, timeZone: string) {
   return schedule.availability.reduce((acc, override) => {
-    // only iff future date override
-    if (!override.date || dayjs.tz(override.date, timeZone).isBefore(dayjs(), "day")) {
+    // only if future date override
+    const currentUtcOffset = dayjs().tz(timeZone).utcOffset();
+    const currentTimeInTz = dayjs().utc().add(currentUtcOffset, "minute");
+
+    if (!override.date || dayjs(override.date).isBefore(currentTimeInTz, "day")) {
```

Comment:
- `override.date` is always 00:00, converting it to a negative timezone resulted in a day in the past
- current time in utc might already be after midnight and therefore on the next day, make sure to use the current time in the user's timezone (`currentTimeInTz`) to compare